src/Centreon/Domain/PlatformTopology/PlatformTopology.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+/*
+ * Copyright 2005 - 2020 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+declare(strict_types=1);
+
+namespace Centreon\Domain\PlatformTopology;
+
+/**
+ * Class designed to retrieve servers to be added using the wizard
+ *
+ */
+class PlatformTopology
+{
+    /**
+     * @var string Server name
+     */
+    private $serverName;
+
+    /**
+     * @var int Server type
+     *      0 = central -> not authorized using the API to register a new server
+     *      1 = poller
+     *      2 = remote server
+     *      3 = map server
+     *      4 = mbi server
+     */
+    private $serverType;
```

Comment:
Please create contants for theses values.

Reply:
this was intended only to document the available values

Reply:
done